wrappers/rust/icicle-core/src/msm/tests.rs

```diff
@@ -31,7 +31,7 @@ where
             .stream = &stream;
         cfg.is_async = true;
         cfg.are_results_on_device = true;
-        C::msm(&scalars, &points, cfg, &mut msm_results.as_slice()).unwrap();
+        let _ = C::msm(&scalars, &points, cfg, &mut msm_results.as_slice()).unwrap();
```

Comment:
Are these `let _ =` really needed in tests?

Reply:
seems it would be safer - and I got also question from @DmytroTym  - so reverted 👍🏻